tests/Fixer/FunctionNotation/PhpdocToParamTypeFixerTest.php

```diff
@@ -428,11 +428,6 @@ function my_foo($foo) {}
             '<?php /** @param Foo[]|Traversable $foo */ function my_foo($foo) {}',
         ];
 
-        yield 'array of object and iterable' => [
-            '<?php /** @param Foo[]|iterable $foo */ function my_foo(iterable $foo) {}',
-            '<?php /** @param Foo[]|iterable $foo */ function my_foo($foo) {}',
-        ];
```

Comment:
Looking at the case's description, it should be:

```suggestion
        yield 'array of object and iterable' => [
            '<?php /** @param object[]|iterable $foo */ function my_foo(iterable $foo) {}',
            '<?php /** @param object[]|iterable $foo */ function my_foo($foo) {}',
        ];

```

I don't know if there's a value in such test case, because it only differs with `object[]` instead of `Foo[]`.

Reply:
For me a test with `For` was already testing objects, but we can have this too, but once again, I cannot apply the changes from here:
![image](https://github.com/user-attachments/assets/fe564179-2f5d-438f-b695-19c337fcd175)